From b66c8c07b9b6ce5c8e44a987fab833b6ca7189d4 Mon Sep 17 00:00:00 2001
From: Tao Guo <tao.guo@amlogic.com>
Date: Wed, 25 Mar 2020 10:30:26 +0800
Subject: [PATCH 04/13] codec_mm: fix codec_mm range [1/1]

PD#SWPL-22078

Problem:
secmem ta protect memory error due to codec_mm range not in white list

Solution:
Adjust codec_mm range to make sure it's in correct range

Verify:
RDK

Change-Id: I09bd85ec7d1d483afa200c8f96c13c2e516da1fc
Signed-off-by: Tao Guo <tao.guo@amlogic.com>
---
 arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts b/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
index a538070b3fd5..5507d829359b 100644
--- a/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
+++ b/arch/arm/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
@@ -114,6 +114,7 @@
 			/* ion_codec_mm max can alloc size 80M*/
 			size = <0x18400000>;
 			alignment = <0x400000>;
+			alloc-ranges = <0x0 0x30000000>;
 			linux,contiguous-region;
 		};
 		/* codec shared reserved */
-- 
2.24.1

